File: 1874_Minimize_Product_Sum_Of_Two_Arrays.py
public class Solution {
    public int MinProductSum(int[] nums1, int[] nums2) {
        Array.Sort(nums1);
        Array.Sort(nums2);
        int sum = 0;
        for (int i = 0; i < nums1.Length; i++) {
            sum += nums1[i] * nums2[nums1.Length - i - 1];
        }
        return sum;
    }
}

File: 1874_Minimize_Product_Sum_of_Two_Arrays.java
class Solution {
    public int minProductSum(int[] nums1, int[] nums2) {
        Arrays.sort(nums1);
        Arrays.sort(nums2);
        int sum = 0;
        for (int i = 0; i < nums1.length; i++) {
            sum += nums1[i] * nums2[nums2.length - i - 1];
        }
        return sum;
    }
}